src/kcas.ml

```diff
@@ -259,6 +324,29 @@ let update_as g loc f (casn, cass) =
       | l, Hit (_loc', state'), r -> update_as g loc f casn state' l r)
   [@@inline]
 
+let attempt (mode : Mode.t) xt =
```

Comment:
`tx` I suspect?

Reply:
Yeah...  It wasn't a typo, but something I failed to rename at some point. 😅

I noticed that already earlier, but didn't bother to change it.  I'll change those.